src/main/java/com/crowdin/client/screenshots/ScreenshotsApi.java

```diff
@@ -127,10 +128,10 @@ public void replaceTags(Long projectId, Long screenshotId, ReplaceTagsRequest re
      * @param projectId    project identifier
      * @param screenshotId screenshot identifier
      * @param request      request object
-     * @return newly created tag
+     * @return newly created tags
      */
-    public ResponseObject<Tag> addTag(Long projectId, Long screenshotId, List<AddTagRequest> request) throws HttpException, HttpBadRequestException {
-        TagResponseObject tagResponseObject = this.httpClient.post(this.url + "/projects/" + projectId + "/screenshots/" + screenshotId + "/tags", request, new HttpRequestConfig(), TagResponseObject.class);
+    public ResponseObject<List<Tag>> addTags(Long projectId, Long screenshotId, List<AddTagRequest> request) throws HttpException, HttpBadRequestException {
+        TagsCreatedResponseObject tagResponseObject = this.httpClient.post(this.url + "/projects/" + projectId + "/screenshots/" + screenshotId + "/tags", request, new HttpRequestConfig(), TagsCreatedResponseObject.class);
```

Comment:
Do we need to create a new class here?

We already have the `TagResponseList` class which is already the same as the tags creation response.

Reply:
fixed